backend/app/services/config_service.py

```
@@ -0,0 +1,168 @@
+        )
+        result = cursor.fetchone()
+
+        db_channel.close()
+
+        if result:
+            preferences_data = json.loads(result[0])
+            return UserPreferences(**preferences_data)
+        else:
+            return UserPreferences()
+
+    def save_user_preferences(self, preferences: UserPreferences):
```

Ruff (0.12.2)

Missing return type annotation for public function `save_user_preferences`

Add return type annotation: `None`

(ANN201)